bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2290,10 +2290,9 @@ namespace {
       Utility::AddToClass(pyclass, "_get__array_interface__", func, METH_NOARGS);
       // Add the dictionary as property to the class so that it updates automatically if accessed.
       // Since we are not able to add a property easily from C++, we do this in Python.
+      // We return early if the module does not have the function to add the property.
+      if (!PyObject_HasAttrString(gRootModule, "_add__array_interface__")) return;
```

Comment:
Nice @stwunsch ! Can you add another line to the comment giving an example of when that can happen (basically, the cppyy case you described in the previous commit). Also I would add an extra line between this comment and the first two comment lines in the method.

Reply:
Fixed!